### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by asifaminisonline**
**Thank you for reviewing my project! On this PR, I completed...**

- Create Rentals and Books classes
- Add associations: the person has many rentals, books have many rentals, rentals have book and person.

_Your review and suggestions on how to improve the project will be greatly appreciated. ⭐_

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```
Comment (m05e5) [inline book.rb]: - [ ] Hi great job so far. As part of the requirements, you are supposed to create a has-many relationship side of Book and Rental (a book has many rentals). So you are supposed to have a method to add rentals in the book class

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 10
+
+    false
+  end
+end
```
Comment (m05e5) [inline person.rb]: - [ ] Hi great job so far. As part of the requirements, you are supposed to create a has-many relationship side of Person and Rental (a person has many rentals). So you are supposed to have a method to add rentals in the person class

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @asifaminisonline , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 2 above)
Comment (asifaminisonline) [reply person.rb]: I don't understand what you want to say, can you provide me some examples

**Comment by asifaminisonline** [comment]
I don't understand what you want to say, can you provide me some examples

File: classroom.rb [hunk 3]
```diff
@@ -1,14 +1,13 @@
 class Classroom
-  attr_accessor :label
-  attr_reader :student
+  attr_accessor :label, :students
```
Comment (AshakaE) [inline classroom.rb]: - [ ] Kindly make `:students` in classroom only readable, we don't want to change students in the class but only add

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @asifaminisonline  🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good job adding the `has-many` update 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @asifaminisonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**